test/support/logger_support.rb

```diff
@@ -1,7 +1,7 @@
 module Runcible
   class Logger
     attr_accessor :message
-    alias_method :debug, :message=
-    alias_method :error, :message=
+    alias debug message=
+    alias error message=
```

Comment:
There are some really minor differences between them mostly stemming from `alias` being a keyword and `alias_method` being a method call. It seems like we use `alias_method` more. I can configure the cop to require `alias_method` but I am tempted just to disable it since they are not 100% interchangeable. More info:

http://blog.bigbinary.com/2012/01/08/alias-vs-alias-method.html
https://ernie.io/2014/10/23/in-defense-of-alias/